lms/djangoapps/courseware/views/views.py

```diff
@@ -226,12 +227,18 @@ def courses(request):
     # Add marketable programs to the context.
     programs_list = get_programs_with_type(request.site, include_hidden=False)
 
+    journal_info = {}
+    journal_info['journals'] = get_journals(request.site)
+    journal_info['journals_root_url'] = get_journals_root_url()
+    journal_info['journal_bundles'] = get_journal_bundles(request.site)
```

Comment:
**DRY:** See comment above about de-duping this code.

Reply:
fixed